src/render/md.jl

```diff
@@ -102,5 +149,69 @@ function writemd(io::IO, m::Meta{:source})
 end
 
 function headermd(io::IO, doc::Metadata, config::Config)
-    print_help(io, config.mdstyle_header, doc.modname)
+    println_mdstyle(io, config.mdstyle_header, doc.modname)
+end
+
+## Docs-specific rendering
+function writemd(io::IO, docs::Docs{:md})
+    println(io, docs.data)
+end
+
+### API-Index ----------------------------------------------------------------------------
+
+type ApiIndex
+    sourcepaths :: Vector{String}
+    modulenames :: Vector{String}
+    entriesmd   :: Vector{EntriesMD}
+
+    ApiIndex() = new(Vector{String}([]), Vector{String}([]), Vector{EntriesMD}([]))
+end
```

Comment:
`String` -> `UTF8String`. Generally speaking, type fields should be concrete rather than abstract types.